scripts/setup-roles.ts

```diff
@@ -0,0 +1,89 @@
+import { ethers } from "hardhat";
+import { SignerWithAddress } from "@nomiclabs/hardhat-ethers/signers";
+
+import { ORIGINATOR_ROLE as DEFAULT_ORIGINATOR_ROLE, ADMIN_ROLE as DEFAULT_ADMIN_ROLE, FEE_CLAIMER_ROLE as DEFAULT_FEE_CLAIMER_ROLE } from "./utils/constants";
+
+export async function main (
+    VAULT_FACTORY_ADDRESS = "0x3A54241cB7801BDea625565AAcb0e873e79C0649",
+    LENDER_NOTE_ADDRESS = "0xdeaBbBe620EDF275F06E75E8fab18183389d606F",
+    BORROWER_NOTE_ADDRESS = "0xaEF48370a5f37CFb760CE44E6cbF986C4DeFF389",
+    ORIGINATION_CONTROLLER_ADDRESS = "0xFE046149bc8830d989A007E81A9D7F8A3b575F36",
+    LOAN_CORE_ADDRESS = "0xd624D1879429A606f54F48B08b56126c3Fe70049",
+    ADMIN_ADDRESS = "0x9b419fd36837558D8A3197a28a5e580AcE44f64F",
+    FEE_CONTROLLER_ADDRESS = "0xE4a1917Ebe8fd2CAFD79799C82aDAa7E81AC6D47",
+    ADMIN_ROLE = DEFAULT_ADMIN_ROLE,
+    FEE_CLAIMER_ROLE = DEFAULT_FEE_CLAIMER_ROLE,
+    ORIGINATOR_ROLE = DEFAULT_ORIGINATOR_ROLE,
+): Promise<void> {
+    const signers: SignerWithAddress[] = await ethers.getSigners();
+    const [admin, borrower] = signers;
+    const deployer = admin;
+
+    if (!LOAN_CORE_ADDRESS) {
+        throw new Error("Must specify LOAN_CORE_ADDRESS in environment!");
+    }
```

Comment:
Some whitespace lines after this `if` checks would improve readability. Artistry!